run

```diff
@@ -515,6 +516,7 @@ lint() {
 
     echo "Code spelling"
     run_command codespell $DIR_TO_RUN --skip="*.onnx,*.min.js,*.min.js.map,Contrastive_learning_Notebook.ipynb,./data" \
+                          --skip="./applications/nvidia_nim/nvidia_nim_nvclip/nvidia_nim.yaml" \
```

Comment:
Please consider using [codelspell ](https://github.com/codespell-project/codespell?tab=readme-ov-file#using-a-config-file)`-config` option or another codespell option to provide the list of excluded files to avoid duplication. I have recently added the ignore-words file to consolidate the false positive words, and the files can also be done similarly.

Reply:
That's what I initially did but it had way too many words to add because of base64 string. I'll think of something else.

Reply:
The last sentence of my comment must have confused you. I meant to say, use a file containing the excluded files instead of listing all of the files (wild card and full path) on the command line.